src/main/java/uk/gov/ons/ctp/response/collection/exercise/service/CollectionExerciseService.java

```diff
@@ -130,6 +163,23 @@ public CollectionExerciseService(
     return collectRepo.findBySurveyIdAndState(surveyId, state);
   }
 
+
+  /**
+   * Find a list of collection exercises associated with a list of surveys from the Collection Exercise Service
+   *
+   * @param surveyIds the survey UUIDS  for which to find collection exercises
+   * @param state Only return collection exercises in this state
+   * @return the associated collection exercises as a dictionary , key is survey id , value is List of collex
```

Comment:
🎨 Same dictionary comment as above (though it might be worth double checking with a java dev before changing it incase I'm wrong).  Also it might be worth having `Value is a list of collection exercises`.  In documentation, calling something what it is instead of an abbreviation can help reduce confusion, in my opinion anyway ^^